AltairZ80/s100_tarbell.c

```diff
@@ -376,6 +384,10 @@ t_stat tarbell_attach(UNIT *uptr, CONST char *cptr)
     t_stat r;
     unsigned int i = 0;
 

```

Comment:
NULL pointer checks in the routines explicitly called by the simh framework (i.e. the DEVICE->attach=tarbell_attach, DEVICE->detach=tarbell_detach, UNIT->action=tarbell_svc) are meaningless.  In each of these contexts, uptr can't be NULL.